internal/resources/ekscluster/resource_nodepool.go

```diff
@@ -64,10 +64,30 @@ var nodepoolSpecSchema = &schema.Schema{
 			},
 			amiTypeKey: {
 				Type:        schema.TypeString,
-				Description: "AMI Type, immutable",
+				Description: "AMI type, immutable",
 				Optional:    true,
 				Computed:    true,
 			},
+			amiInfoKey: {
+				Type:        schema.TypeList,
+				Description: "AMI info for the nodepool if AMI type is specified as CUSTOM",
+				Optional:    true,
+				MaxItems:    1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						amiIDKey: {
+							Type:        schema.TypeString,
+							Description: "ID of the AMI to be used",
+							Optional:    true,
```

Comment:
Do we have a default value for `amiID` and `overrideBootstrap`?

Reply:
No, they are specific to the AMIs that the user wants to use.